nexus/src/app/background/init.rs

```diff
@@ -778,7 +778,8 @@ impl BackgroundTasksInitializer {
                 "detect if region snapshots need replacement and begin the \
                 process",
             period: config.region_snapshot_replacement_start.period_secs,
-            task_impl: Box::new(RegionSnapshotReplacementDetector::new(
+            // XXX temporarily disabled
```

Comment:
Is there an issue this could reference on top of `XXX`? I don't feel strongly about this if it's going to be reenabled "soon" though, up to you.